fido2/ctaphid.c

```diff
@@ -745,16 +741,21 @@ uint8_t ctaphid_custom_command(int len, CTAP_RESPONSE * ctap_resp, CTAPHID_WRITE
 
         case CTAPHID_GETVERSION:
             printf1(TAG_HID,"CTAPHID_GETVERSION\n");
-            wb->bcnt = 3;
+            wb->bcnt = 4;
             ctap_buffer[0] = SOLO_VERSION_MAJ;
             ctap_buffer[1] = SOLO_VERSION_MIN;
             ctap_buffer[2] = SOLO_VERSION_PATCH;
```

Comment:
Correct me if I am wrong, but I believe any site can call this command. Is it button-guarded? If not, then it would be nice to add such user confirmation to avoid providing additional data for fingerprinting.

Reply:
Websites cannot issue HID commands through FIDO API.  It is potentially additional info for fingerprinting.  It's main use is to silently tell if the device needs a firmware update, so requiring a button press would hamper the UX.  I'm for keeping it silent.

Reply:
Let's move the further conversation to https://github.com/solokeys/solo/issues/339.